packages/landing-app/next.config.mjs

```diff
@@ -62,6 +62,11 @@ const nextConfig = {
     return config;
   },
   reactStrictMode: true,
+  swcMinify: true,
```

Comment:
Esta configuração utiliza o SWC (Speedy Web Compiler) do Next.js para minificação.

Impacto esperado nos Web Vitals:
Melhora o LCP (Largest Contentful Paint) por reduzir o tempo de download dos bundles
Melhora o FID (First Input Delay) por ter código mais otimizado para carregar
Reduz o TTI (Time to Interactive) por ter bundles mais leves